guides/source/form_helpers.md

```diff
@@ -76,10 +78,6 @@ This will generate the following HTML:
 
 TIP: For every form input, an ID attribute is generated from its name (`"q"` in above example). These IDs can be very useful for CSS styling or manipulation of form controls with JavaScript.
 
-Besides `text_field_tag` and `submit_tag`, there is a similar helper for _every_ form control in HTML.
-
-IMPORTANT: Always use "GET" as the method for search forms. This allows users to bookmark a specific search and get back to it. More generally Rails encourages you to use the right HTTP verb for an action.
```

Comment:
Why is this useless?

Reply:
I just returned this. I removed "Always" from the beginning of the sentence since it sounds like an obligation with this word.